src/test/java/seedu/trackbeau/testutil/EditPersonDescriptorBuilder.java

```diff
@@ -72,12 +78,48 @@ public EditPersonDescriptorBuilder withAddress(String address) {
     }
 
     /**
-     * Parses the {@code tags} into a {@code Set<Tag>} and set it to the {@code EditPersonDescriptor}
+     * Sets the {@code Skin Type} of the {@code EditPersonDescriptor} that we are building.
+     */
+    public EditPersonDescriptorBuilder withSkinType(String skinType) {
```

Comment:
Do we need to rename this class to EditCustomerDescriptorBuilder? Or can this class be a superclass for both Customer and Staff?

Reply:
superclass sounds good

Reply:
not going to implement superclass for now